src/shared/components/uploadImage/uploadImage.component.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { contentfulConfig } from '../../services/api/contentful';
+import { getEnvironment, uploadImage } from '../adminPanelTable/adminPanelTable.utils';
+import { EditedEntry } from '../../../routes/adminPanel/adminPanel.types';
+import { StyledFileInput } from './uploadImage.styles';
+
+interface UploadImageProps {
+  editedEntry: EditedEntry;
+}
+
+export const UploadImage: React.FC<UploadImageProps> = ({ editedEntry }: UploadImageProps) => {
+  const [image, setImage] = useState<File | null>(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (!event.target.files?.[0]) return null;
+    setImage(event.target.files![0]);
+  };
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setLoading(true);
+    setError('');
+
+    if (!image) {
+      alert('File is null!');
```

Comment:
Maybe "The file is invalid" is better? A lot of people don't know what "null" is